Solutions/JuniperIDP/Parsers/Juniper_IDP.txt

```diff
@@ -0,0 +1,93 @@
+// Usage Instruction : 
```

Comment:
Remove the all the column_ifexists, which are not necessary. Use project-rename when no transformation is needed.

Reply:
Done